examples/rsp-cra-18/src/Completed.tsx

```diff
@@ -0,0 +1,38 @@
+import Delete from '@spectrum-icons/workflow/Delete';
+import {AlertDialog, DialogTrigger, ActionButton} from '@adobe/react-spectrum'
+import {Checkbox} from '@adobe/react-spectrum'
+import {Flex} from '@adobe/react-spectrum'
+
+
+function Completed(props: {completed: {id: number; task: string}[]; onDelete: any}){
+
+    const elements = props.completed.map(item => (
+        <Checkbox isDisabled key={item.id}>{item.task}</Checkbox>
```

Comment:
design feedback: Maybe these would be better off as readonly + checked Checkbox?